site/en/docs/_index.yaml

```diff
@@ -115,12 +115,18 @@ landing_page:
       path: /configure/toolchain-resolution
     - heading: Code coverage with Bazel
       path: /configure/coverage
-    - heading: Best practices
+    - heading: Optimizing Bazel: Best practices
```

Comment:
the title needs to be quoted here, otherwise the `:` will be mistaken as YAML syntax element

Reply:
Done.